cicd/gitlab/dot.gitlab-ci.yml

```
@@ -51,6 +49,11 @@ include:
   - local: cicd/gitlab/parts/license.gitlab-ci.yml
   # Flat file checks
   - local: cicd/gitlab/parts/flatfile.gitlab-ci.yml
+  # Genomio docker deploy (on release tag creation)
+  - local: cicd/gitlab/parts/dockerbuild.genomio.gitlab-ci.yml
+    rules:
+      - if: '$CI_COMMIT_TAG =~ /v[0-9]\.[0-9]\.[0-9]$/'
```

```suggestion
      - if: '$CI_COMMIT_TAG =~ /v[0-9]+\.[0-9]+\.[0-9]+$/'
```
Having the "hopeful" thought if we reach to a 10+ (sub)version.